packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -54,6 +55,46 @@ class _SimpleExpansionPanelListTestWidgetState extends State<SimpleExpansionPane
   }
 }
 
+Key headerKey = UniqueKey();
```

Comment:
Instead of having this as a global it would probably be more appropriate to have the  ExpansionPanelListSemanticsTest constructor take in an actual key, which is then passed in by the individual tests.